src/main/assembly/dist/md/ddm/ddm.xsd

```diff
@@ -212,66 +215,38 @@
     </xs:element>
 
     <!-- =================================================================================== -->
-    <xs:element name="relation" substitutionGroup="dc:relation">
+    <xs:element name="linkedRelation" substitutionGroup="dc:relation" type="ddm:LinkedRelationType">
```

Comment:
I assume this is a kind of 'abstract' element? Or can we actually use `<ddm:linkedRelation/>` in the DDM?
In that case, doesn't this break backwards compatibility, given that you remove the `<ddm:relation/>` type?